spec/models/like_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```

Comment:
- [ ] I suggest that any custom methods in the models should have a corresponding unit test so please ensure that you add a test for the `increment_post_likes_counter` method in your like model :pray: